.gitignore

```diff
@@ -1,3 +1,4 @@
+tools/
```

Comment:
This should not be ignored, there are still some tools used during the build.

Reply:
Cause the other files are already under source (and packages.config is force added) this should not harm if we plan to download the build dependencies from nuget anyway